xml/tuning_kexec.xml

```diff
@@ -1256,19 +1253,19 @@ PID: 9446   TASK: ffff88003a57c3c0  CPU: 1   COMMAND: "bash"
   </para>
 
   <para>
-   Kernel dumps are usually huge and contain many pages that are not necessary
+   Kernel dumps are large and contain many pages that are not necessary
    for analysis. With <option>KDUMP_DUMPLEVEL</option> option, you can omit
    such pages. The option understands numeric value between 0 and 31. If you
-   specify <replaceable>0</replaceable>, the dump size will be largest. If you
-   specify <replaceable>31</replaceable>, it will produce the smallest dump.
+   specify <replaceable>0</replaceable>, the dump size is the largest. If you
+   specify <replaceable>31</replaceable>, it produces the smallest dump.
    For a complete table of possible values, see the manual page of
    <command>kdump</command> (<command>man 7 kdump</command>).
   </para>
 
   <para>
-   Sometimes it is very useful to make the size of the kernel dump smaller. For
-   example, if you want to transfer the dump over the network, or if you need
-   to save some disk space in the dump directory. This can be done with
+   Sometimes it is useful to make the size of the kernel dump smaller. For
+   example, to transfer the dump over the network, or if you need
+   to save disk space in the dump directory. This can be done with
```

Comment:
```suggestion
   Sometimes it is useful to make the size of the kernel dump smaller. For
   example, you can do so to transfer the dump over the network or to 
   save disk space in the dump directory. This can be done with
```